### ISSUE TITLE
What's the latent and sample mean

### ISSUE THREAD

**Issue by ligenjie** [IceClear/StableSR#7]
I try to train the model,but the dataloader have the input of gt,lq,latent and sample.I don't understand how to get latent and samples ?  how to get my custom data's latent and samples

**Comment by IceClear** [comment]
Hi
If you mean training the CFW, you first need to generate samples using the trained diffusion model.
You need to save the output latent code before VQGAN decoder as 'latent' and the generated images after VQGAN decoder as 'samples'.
'Samples' are just for visualization. We actually do not use it for training but just make sure the generated data is correct.

**Comment by ligenjie** [comment]
Thank u~ I have another question,if I want to train my custom model using pretrained model. If the train pipline like latent diffusion model? I must first train the autoencoder then train the diffusion model? Or the autoencoder’s performance and accuracy are good enough that I can  skip training first stage autoencoder model and direct train the second stage diffusion model

**Comment by IceClear** [comment]
I am not sure what is your goal of training such a model, so I just explain the situation of StableSR, i.e., we did not fine-tune the VQGAN architecture.
I suggest you to take a quick test on your data to see if VQGAN reconstructs well.

**Comment by ligenjie** [comment]
Thank u ~
